src/include/migraphx/register_op.hpp

```diff
@@ -33,15 +33,33 @@
 namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 
+// unregister all ops for specified target, useful when unloading dynamically plugged-in target lib
+void unregister_op(const std::string& op_name);
+
+struct op_handler
+{
+    operation op;
```

Comment:
This variable is never used.

Reply:
It is used later as when we call `register_op()`